common/utils.js

```diff
@@ -94,7 +95,7 @@ Workflow: ${github_1.context.workflow}
 
 Error: ${message}
 
-Issue: ${ping ? `${github_1.context.repo.owner}/${github_1.context.repo.repo}#` : ''}${github_1.context.issue.number}
+Issue: ${ping ? `${github_1.context.repo.owner}/${github_1.context.repo.repo}#` : ''}${(_a = github_1.context.issue) === null || _a === void 0 ? void 0 : _a.number}
```

Comment:
I guess ideally these built files wouldn't be checked in

Reply:
Not ideal, but the alternative is installing a typescript transpiler of some sort on every action run, which amounts to quite a lot of wasted compute.